projects/1.16/assets/iron-furnaces/ironfurnaces/lang/zh_cn.json

```
@@ -1,125 +1,144 @@
-  "ironfurnaces.update.buttonOptions": "点击：更改日志，Shift点击：下载！(在浏览器)",
-  "ironfurnaces.update.failed": "[{\"text\":\"更多熔炉\"},{\"text\":\"检测更新\",\"color\":\"dark_green\"},{\"text\":\"检测更新失败！查看日志以获取更多信息！\",\"color\":\"none\"}]"
+    "_comment": "Items",
+    "item.ironfurnaces.upgrade_iron": "熔炉升级：石-铁",
+    "item.ironfurnaces.upgrade_gold": "熔炉升级：铁-金",
+    "item.ironfurnaces.upgrade_diamond": "熔炉升级：金-钻石",
+    "item.ironfurnaces.upgrade_emerald": "熔炉升级：钻石-绿宝石",
+    "item.ironfurnaces.upgrade_obsidian": "熔炉升级：绿宝石-黑曜石",
+    "item.ironfurnaces.upgrade_crystal": "熔炉升级：钻石-水晶",
+    "item.ironfurnaces.upgrade_obsidian2": "熔炉升级：水晶-黑曜石",
+    "item.ironfurnaces.upgrade_netherite": "熔炉升级：黑曜石-下界合金",
+    "item.ironfurnaces.item_heater": "无线热能接收器",
+    "item.ironfurnaces.augment_blasting": "熔炉组件：高炉",
+    "item.ironfurnaces.augment_smoking": "熔炉组件：烟熏炉",
+    "item.ironfurnaces.augment_speed": "熔炉组件：速度升级",
+    "item.ironfurnaces.augment_fuel": "熔炉组件：燃料升级",
+    "item.ironfurnaces.augment_redstone": "熔炉组件：红石控制",
+    "item.ironfurnaces.gui": "熔炉组件：物品栏配置",
+    "item.ironfurnaces.item_spooky": "熔炉装饰：骇人南瓜",
+    "item.ironfurnaces.item_xmas": "熔炉装饰：圣诞礼物",
+    "item.ironfurnaces.rainbow_core": "彩虹核心",
+    "item.ironfurnaces.rainbow_plating": "彩虹外壳",
+    "item.ironfurnaces.rainbow_coal": "彩虹煤炭",
+    "item.ironfurnaces.upgrade_copper": "熔炉升级：石-铜",
+    "item.ironfurnaces.upgrade_silver": "熔炉升级：铜-银",
+    "item.ironfurnaces.upgrade_silver2": "熔炉升级：铁-银",
+    "item.ironfurnaces.upgrade_iron2": "熔炉升级：铜-铁",
+    "item.ironfurnaces.upgrade_gold2": "熔炉升级：银-金",
+    "item.ironfurnaces.item_copy": "熔炉配置链接器",
+    "item.ironfurnaces.item_linker": "彩虹连接器",
+
+    "_comment": "TAB",
+    "itemGroup.ironfurnaces": "更多熔炉",
+
+    "_comment": "Blocks",
+    "block.ironfurnaces.iron_furnace": "铁熔炉",
+    "block.ironfurnaces.gold_furnace": "金熔炉",
+    "block.ironfurnaces.diamond_furnace": "钻石熔炉",
+    "block.ironfurnaces.emerald_furnace": "绿宝石熔炉",
+    "block.ironfurnaces.obsidian_furnace": "黑曜石熔炉",
+    "block.ironfurnaces.crystal_furnace": "水晶熔炉",
+    "block.ironfurnaces.netherite_furnace": "下界合金熔炉",
+    "block.ironfurnaces.copper_furnace": "铜熔炉",
+    "block.ironfurnaces.silver_furnace": "银熔炉",
+    "block.ironfurnaces.million_furnace": "彩虹熔炉",
+    "block.ironfurnaces.heater": "无线热能供应器",
+
+    "_comment": "ATM",
+    "block.ironfurnaces.allthemodium_furnace": "ATM熔炉",
+    "block.ironfurnaces.vibranium_furnace": "振金熔炉",
+    "block.ironfurnaces.unobtainium_furnace": "难得素熔炉",
+    "container.ironfurnaces.allthemodium_furnace": "ATM熔炉",
+    "container.ironfurnaces.vibranium_furnace": "振金熔炉",
+    "container.ironfurnaces.unobtainium_furnace": "难得素熔炉",
+    "item.ironfurnaces.upgrade_allthemodium": "熔炉升级：下界合金-ATM",
+    "item.ironfurnaces.upgrade_vibranium": "熔炉升级：ATM-振金",
+    "item.ironfurnaces.upgrade_unobtainium": "熔炉升级：振金-难得素",

```

```suggestion
    "item.ironfurnaces.upgrade_allthemodium": "熔炉升级：下界合金->ATM",
    "item.ironfurnaces.upgrade_vibranium": "熔炉升级：ATM->振金",
    "item.ironfurnaces.upgrade_unobtainium": "熔炉升级：振金->难得素",
```
ATM6里有合金叫`atm-vibranium` `vibranium-unobtainium`